sdk/identity/identity/test/internal/node/msalClient.spec.ts

```diff
@@ -97,24 +127,6 @@ describe("MsalClient", function () {
       sandbox = sinon.createSandbox();
     });
 
-    it("is supported", async function () {
-      const scopes = ["https://vault.azure.net/.default"];
-      const clientSecret = process.env.AZURE_CLIENT_SECRET!;
-
-      const client = msalClient.createMsalClient(clientId, tenantId);
-
-      sandbox
-        .stub(ConfidentialClientApplication.prototype, "acquireTokenByClientCredential")
-        .resolves({
-          accessToken: "token",
-          expiresOn: new Date(Date.now() + 3600 * 1000),
```

Comment:
Karishma and I chatted about adding validation for this, which I will do in a follow-up PR. Just keeping myself honest 😄